Dockerfile.antidote

```diff
@@ -0,0 +1,113 @@
+ARG ELIXIR_VERSION=1.13.4
+ARG OTP_VERSION=24.3
+ARG DEBIAN_VERSION=bullseye-20210902-slim
+
+ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
+ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"
+
+FROM ${BUILDER_IMAGE} AS builder
+
+LABEL maintainer="dev@vaxine.io"
+
+RUN apt-get update -y && apt-get install -y build-essential git curl \
+    && apt-get clean && rm -f /var/lib/apt/lists/*_*
+
+RUN mix local.hex --force                                                                                                                                                
+RUN mix local.rebar --force  
+
+WORKDIR /app
+
+ENV CI=true
+
+COPY rebar.config /app/
+COPY rebar.lock /app/
+
+RUN /root/.mix/rebar3 compile
+
+COPY Makefile /app/
+COPY config/ /app/config/
+COPY apps/ /app/apps/
+
+RUN /root/.mix/rebar3 compile
+RUN /root/.mix/rebar3 release
+
+FROM ${RUNNER_IMAGE} AS runner_setup
+
+RUN apt-get update -y && apt-get install -y libstdc++6 openssl libncurses5 locales \
+  && apt-get clean && rm -f /var/lib/apt/lists/*_*
+
+# Set the locale
+RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && locale-gen
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+WORKDIR "/app"
+RUN chown nobody /app
```

Comment:
Not sure why do you need this here. Commonly if you want to build something in docker on your local machine - you don't want to mess with the ownership, but rather you want to keep file permissions, and just mount local user in the container.

Reply:
Ok, we clearly have an intention mismatch on this. I need to be able to build production-ready Antidote docker container either locally to run some code against that or, mainly, in CI. I intend for images built against this dockerfile to be ran in our k8s and sometimes locally for close-to-production tests when you need to check something particular.

I do not intend for this image to be an "active development with volumes, local user and stuff like that" at all. I personally don't see a need in such images, but if you do - we can write a dockerfile for that. I'm not against that at all, but that's a completely different scope from this

Reply:
Yeah, that's a pretty fair case